package.json

```diff
@@ -53,7 +53,10 @@
     "serve:no-open": "ionic serve --no-open",
     "sync": "ionic cap sync",
     "tdex:clean": "bash scripts/tdexd-clean.sh",
-    "tdex:run": "curl https://raw.githubusercontent.com/TDex-network/tdex-sdk/master/scripts/tdexd-run -o tdex-run && bash tdex-run && rm tdex-run",
+    "tdex:connect": "docker exec -it tdexd tdexdconnect --out url",
+    "tdex:run": "bash scripts/tdexd-run",
+    "tdex:run-init": "bash scripts/tdexd-run-initialized.sh",
```

Comment:
whats the difference from this and the `tdex:run-initialized`?

Shall we centralize a script in single place?

Reply:
Running daemon through Docker I need `-v "$(pwd)/tdexd:/home/user/.tdex-daemon"` but CI and you guys needs `-v "$(pwd)/tdexd:/.tdex-daemon"`. So I kept both scripts. I still don't know why we don't have the same config.

Reply:
🤔 🤔 🤔  the docker file should be the same